ql/src/test/queries/clientpositive/materialized_view_create_rewrite_8.q

```diff
@@ -0,0 +1,97 @@
+-- Test Incremental rebuild of materialized view without aggregate when a source table is insert only.
+
+SET hive.vectorized.execution.enabled=false;
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.strict.checks.cartesian.product=false;
```

Comment:
Why do the properties below matter for the incremental rebuild?
```
hive.vectorized.execution.enabled=false
hive.strict.checks.cartesian.product=false
```

Reply:
Removed these.